packages/legacy/core/App/localization/en/index.ts

```diff
@@ -676,6 +677,7 @@ const translation = {
     "NoInfoShared": "No information was shared",
     "YourInfo": "Your information was not shared",
     "YouCantRespond": "You can't respond due to the following reasons. Please address them before continuing.",
+    "CredentailNotInWallet": "This credential is not present in your wallet.",
```

Comment:
Should be `CredentialNotInWallet` I think

Reply:
done